### DISCUSSION TITLE
is `github.community` gone?

### DISCUSSION THREAD

**Discussion by KisaragiEffective** [community/community#31131] [Other Feature Feedback, Questions, & Ideas] [answered]
The domain redirects here, I suppose the site is migrated with this forum?

**Comment by melvincwng** [comment, marked as answer]
@KisaragiEffective , yes you are absolutely right.

The original github.community discourse forum has already been closed weeks ago. Any attempts to go that site will be redirected over here.

Hope this answers your question 😁

**Reply by KisaragiEffective** [reply]
Thanks for quick answer :)

**Reply by melvincwng** [reply]
Pleasure in helping ;)
